.github/workflows/deploy-2-start.yml

```diff
@@ -54,6 +60,14 @@ jobs:
             ${{ secrets.HOST }}
             ${{ secrets.HOST_DATA }}
 
+      - name: Prerelease Modulefile Name
+        if: inputs.type == 'prerelease'
+        # Modifies the name of the prerelease modulefile to the
+        # `pr<number>-<commit>` style. For example, `access-om3/pr12-2`
```

Comment:
In the PR text I thought the name was of the form `access-om3-pr12-2`

Reply:
In the PR description it has:
> For example, `access-om3/pr12-2`.